docs/element-templates/README.md

```diff
@@ -171,13 +182,15 @@ As seen in the example the important attributes in a property definition are:
 * `type`: Defining the visual appearance in the properties panel (may be any of `String`, `Text`, `Boolean`, `Dropdown` or `Hidden`)
 * `value`: An optional default value to be used if the property to be bound is not yet set
 * `binding`: Specifying how the property is mapped to BPMN or Camunda extension elements and attributes (may be any of `property`, `camunda:property`, `camunda:inputParameter`, `camunda:outputParameter`, `camunda:in`, `camunda:out`, `camunda:executionListener`, `camunda:field`)
-* `constraints`: A list of editing constraints to apply to the template
+* `constraints`: A list of editing constraints to apply to the template (note that this configuration only works if a custom `type` is configured)
 
 
 #### Types
 
 The input types `String`, `Text`, `Boolean`, `Dropdown` and `Hidden` are available. As seen above `String` maps to a single-line input, `Text` maps to a multi-line input.
 
+If `type` is omitted the default UI component will be rendered for the respective binding. This behavior is currently only supported for `camunda:inputParameter` and `camunda:outputParameter`.
```

Comment:
Technically we also have default components for 

* `property`, `camunda:property`, `camunda:in`, `camunda:in:businessKey`, `camunda:out`, `camunda:field` -> String
* `camunda:executionListener` -> Hidden

Cf. https://github.com/bpmn-io/bpmn-js-properties-panel/issues/358

It's just a question of whether we want to publicly speak about this or it is more an internal thing.

Reply:
True => I will include this as side note.

Reply:
I am not sure whether it's confusing when we declare here

> only supported for `camunda:inputParameter` and `camunda:outputParameter`

but saying below it's possible for other types